addons/pos_restaurant/static/src/js/Screens/TicketScreen.js

```diff
@@ -89,7 +89,7 @@ odoo.define('pos_restaurant.TicketScreen', function (require) {
                     }
 
                     if (!amount) {
-                        await this.setNoTip();
+                        await this.setNoTip(serverId);
```

Comment:
this looks like the kind of change I would expect a test... but no idea what the POS team would do